gradle/any/dependencies.gradle

```diff
@@ -9,7 +9,7 @@ repositories {
     // the local maven repository and build the TDS with those. In the netCDF-java travis build script, 
     // we use sed to uncomment the following line so that the tds build uses those local snapshot artifacts in its
     // build.
-    //mavenLocal()
+    // mavenLocal()
```

Comment:
Can you remove the space here? I need to update my regex in sed (used in the travis script in the netcdf-java repo) to be ok with a whitespace if one sneaks in, but right now, it does not work.